### ISSUE TITLE
[Dell] [STUDIO 1640] Phoenix BIOS 5-bigit System Disabled Lock Code incorrect key

### ISSUE THREAD

**Issue by phillipremaker** [bacher09/pwgen-for-bios#275]
### Please specify your environment

* **OS**: Windows 10
* **Browser**: Chrome 129
* **website domain**: ([bios-pw.org](https:///bios-pw.org/)) 
* **website version**: 5d2caca

### Expected behavior
When entering a 5 digit Phoenix BIOS "System Disabled," I expect a valid unlock code, the same as generated by pwgen-5dec.py in Dogbert's repository.

For [02793], the correct results are:
Brute forcing passwords...
Generic Phoenix BIOS:          ptrq
HP/Compaq Phoenix BIOS:        zzkos
FSI Phoenix BIOS (generic):    4727548
FSI Phoenix BIOS ('L' model):  4757274
FSI Phoenix BIOS ('P' model):  1562661
FSI Phoenix BIOS ('S' model):  534634
FSI Phoenix BIOS ('X' model):  5892966

### Actual behavior
The website generates incorrect results:

For [02793], the rebsite generates:
Generic Phoenix	bieegs
HP/Compaq Phoenix BIOS	ysaa
Fujitsu-Siemens Phoenix	7456372
Fujitsu-Siemens (model L) Phoenix	8882525
Fujitsu-Siemens (model P) Phoenix	74146
Fujitsu-Siemens (model S) Phoenix	69637
Fujitsu-Siemens (model X) Phoenix	779947

### Steps to reproduce the issue
Enter the code [02793], compare results to https://github.com/dogbert/bios-pwgen/pwgen-5dec.py 

- Please be as specific as possible

### Additional information

Attach additional information that could help to reproduce the bug (screenshot, browser console
logs, etc).

**Comment by bacher09** [comment]
@phillipremaker Have you tried running this script **multiple times**? I mean `pwgen-5dec.py`, it's a random-based, it produces each time a [different output](https://github.com/dogbert/bios-pwgen/blob/master/pwgen-5dec.py#L75), same goes for the website as well, you can just click `Get password` a few more times and each time you'll get a different password. What is crucial is that a hash of an output produces the same number, and for both cases it's the same. It even has a unit-tests that checks results of a hash function for that [particular algorithm](https://github.com/bacher09/pwgen-for-bios/blob/main/src/keygen/phoenix.spec.ts).

The same goes for different passwords, it isn't necessary should be the same generated passwords, although it depends on a firmware.

Anyhow, I'm closing this for now.

**Comment by bacher09** [comment]
A few samples that show a different result that you could get.
![image](https://github.com/user-attachments/assets/7d88a984-1621-43fa-a3a7-e19f1325217b)
![image](https://github.com/user-attachments/assets/ff6ed2c5-8ecd-4ddf-898c-ea74b0d7b8e0)


Both results absolutely equivalent, the same may apply to some other algorithms, some were researched independently and have slightly different generators, etc. The fact that passwords aren't matched isn't necessary means that there is a bug.

**Comment by phillipremaker** [comment]
Thanks for the fast reply! I could have sworn that it did not work for me before, but once I went through the whole process again of setting a new password and then using the website... it worked!!

I have no idea what I did wrong the first time.

Once again, thanks for actively maintaining this site. It has saved me on more than one occassion. This PC can now be put back to use for someone who would not otherwise have a laptop. 

Thanks to you!
